src/gui/application.cpp

```diff
@@ -621,8 +622,11 @@ void Application::parseOptions(const QStringList &options)
             if (it.hasNext() && !it.peekNext().startsWith(QLatin1String("--"))) {
                 _userEnforcedLanguage = it.next();
```

Comment:
Please check wehter the selected language is in the list of available languages and fail if not.

Reply:
Done. It makes little sense, though. A locale like `en_US` will still lead to an English translation to be loaded even though only an `en` file is bundled. Therefore, I initially didn't want to implement this. Perhaps we can just print a warning instead?

Reply:
Actually, it looks like `en_US` leads to an English translation (that would be correct), but `nl_NL` leads to an English one as well (should be `nl`). Therefore, I think validating the input as you suggested is the right way to go.

Reply:
Strike that, I had configured the language to English in the settings. `nl_NL` works fine for `--language`, even though it's not available in the list.